<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException" namespace="Symfony\Component\HttpKernel\Exception" name="UnauthorizedHttpException" abstract="false" final="false" start="19" end="35">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/http-kernel/Exception" file="UnauthorizedHttpException.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/symfony/http-kernel/Exception/UnauthorizedHttpException.php" size="979" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="70cc12e65b4e805ad579ff55e453c4e8604e3509" relative="vendor/symfony/http-kernel/Exception/UnauthorizedHttpException.php"/>
  <docblock>
    <description compact="UnauthorizedHttpException."/>
    <author value="Ben Ramsey &lt;ben@benramsey.com&gt;"/>
  </docblock>
  <extends full="Symfony\Component\HttpKernel\Exception\HttpException" namespace="Symfony\Component\HttpKernel\Exception" name="HttpException"/>
  <constructor name="__construct" start="29" end="34" abstract="false" final="false" static="false" visibility="public">
    <docblock>
      <description compact="Constructor."/>
      <param description="WWW-Authenticate challenge string" type="string" variable="$challenge"/>
      <param description="The internal exception message" type="string" variable="$message"/>
      <param description="The previous exception" type="object" variable="$previous">
        <type full="\Exception" namespace="" name="Exception"/>
      </param>
      <param description="The internal exception code" type="int" variable="$code"/>
    </docblock>
    <parameter name="challenge" byreference="false" type="{unknown}"/>
    <parameter name="message" byreference="false" type="{unknown}" default="" constant="null"/>
    <parameter name="previous" byreference="false" type="{unknown}" default="" constant="null">
      <type full="Exception" namespace="" name="Exception"/>
    </parameter>
    <parameter name="code" byreference="false" type="integer" default="0"/>
  </constructor>
</class>
